### ISSUE TITLE
how to compare embeddings/logits after distillation?

### ISSUE THREAD

**Issue by chuangzhidan** [MinishLab/model2vec#171]
wanna know how much performance loss  i will get from distillation

**Comment by chuangzhidan** [comment]
> Hello,
> 
> you should use an extrinsic task, i.e., the task you'd like to use your model for. There is no real other way to know how much performance you will lose.
> 
> Stéphan

ok ,thank u
